<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像失真校正</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="srcDisplay_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>413</width>
      <height>284</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="srcDisplay_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>413</width>
      <height>284</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="display_src_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示原始图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="img_match_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>图像配准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="img_correct_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>图像校正</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_src">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_src_y">
        <item>
         <widget class="QLineEdit" name="lineEdit_src_x_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_x_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_x_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_x_4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_src_x">
        <item>
         <widget class="QLineEdit" name="lineEdit_src_y_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_y_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_y_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_src_y_4"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_target">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_target_x">
        <item>
         <widget class="QLineEdit" name="lineEdit_target_x_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_x_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_x_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_x_4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_target_y">
        <item>
         <widget class="QLineEdit" name="lineEdit_target_y_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_y_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_y_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_target_y_4"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
